roles/ceph-rgw/tasks/common.yml

```diff
@@ -41,3 +41,13 @@
   when:
     - cephx | bool
     - item.item.copy_key | bool
+
+- name: copy SSL certificate & key data to certificate path
+  copy:
+    content: "{{ radosgw_frontend_ssl_certificate_data }}"
+    dest: "{{ radosgw_frontend_ssl_certificate }}"
+    owner: root
+    group: root
+    mode: 0440
```

Comment:
I'm not sure that the ceph user will be able to read that file

Reply:
in our deployment we have the combined certificate file owned by root, group root with permissions of 640 which works. I didn't think taking user write permission away would impact ceph's ability to read the file. Do you suggest the file ownership should be ceph?

Reply:
I'm curious to understand how the ceph user is able to read this certificate if only root user and root group are allowed.
By default the ceph user is only in the ceph group.

```console
[root@1d36b5fd076d ~]# echo DEADBEAF > /etc/radosgw.pem
[root@1d36b5fd076d ~]# chmod 440 /etc/radosgw.pem
[root@1d36b5fd076d ~]# ls -hl /etc/radosgw.pem 
-r--r----- 1 root root 11 Nov 13 20:33 /etc/radosgw.pem 
[root@1d36b5fd076d ~]# su - ceph
[ceph@1d36b5fd076d ~]$ cat /etc/radosgw.pem 
cat: /etc/radosgw.pem: Permission denied
```

Reply:
I've corrected user & group permissions here for the PR. 
I'll investigate further how our setup is working; deployment was pre Luminous and may have been subsequently "handcrafted"...

Reply:
Any further changes needed to make this acceptable/ok for merging?